config/routes.rb

```diff
@@ -0,0 +1,15 @@
+Rails.application.routes.draw do
+  devise_for :users
+  # get    '/posts'          => 'posts#index'
+  # get    '/posts/new'      => 'posts#new'
+  # get    '/posts/:id'      => 'posts#show'
+  # post   '/posts'          => 'posts#create'
+  # get    '/posts/:id/edit' => 'posts#edit'
+  # put    '/posts/:id'      => 'posts#update'
+  # delete '/posts/:id'      => 'posts#destroy'
+  root to: 'posts#index'
+
+  resources :posts do
+    resources :comments
+  end
```

Comment:
I think I left a comment last time about only using the nested `comments` for a few actions like `create` and `index`, and then a not-nested `comments` for everything else so you don't get redundant IDs in your URLs (e.g. `/posts/32/comments/98`. I'd like to see you try that on your project!